pkg/login/poll.go

```diff
@@ -15,16 +15,19 @@ import (
 const maxAttemptsDefault = 2 * 60
 const intervalDefault = 1 * time.Second
 
-type pollAPIKeyResponse struct {
-	Redeemed           bool   `json:"redeemed"`
-	AccountID          string `json:"account_id"`
-	AccountDisplayName string `json:"account_display_name"`
-	APIKey             string `json:"testmode_key_secret"`
-	PublishableKey     string `json:"testmode_key_publishable"`
+// PollAPIKeyResponse returns the data of the polling client login
+type PollAPIKeyResponse struct {
+	Redeemed               bool   `json:"redeemed"`
+	AccountID              string `json:"account_id"`
+	AccountDisplayName     string `json:"account_display_name"`
+	TestModeAPIKey         string `json:"testmode_key_secret"`
+	TestModePublishableKey string `json:"testmode_key_publishable"`
 }
 
 // PollForKey polls Stripe at the specified interval until either the API key is available or we've reached the max attempts.
-func PollForKey(pollURL string, interval time.Duration, maxAttempts int) (string, string, *Account, error) {
+func PollForKey(pollURL string, interval time.Duration, maxAttempts int) (PollAPIKeyResponse, *Account, error) {
```

Comment:
Will defer to Brandur, but I think it's a bit more idiomatic to return a pointer (`* PollAPIKeyResponse`) and return `nil` instead of an "empty" struct when returning an error.

Reply:
oh, yes, I always do that. Changing now!